### PR TITLE
Create Cyanic.json

### PR DESCRIPTION
**Pull request by Cyanic76**

### CODE REVIEW

**Review by AbubakarKang — COMMENTED** [review]
Your URL record must contain a protocol. So instead of `cyanic.tk`, it should be `https://cyanic.tk/`

File: domains/Cyanic.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Cyanic's website (link to cyanic.tk)",
+  "repo": "https://github.com/Cyanic76/Cyanic76",
+  "owner": {
+    "username": "Cyanic76",
+    "email": "git@cyanic.tk"
+  },
+  "record": {
+    "URL": "cyanic.tk"
```
Comment (MaskDuck) [inline domains/Cyanic.json:9]: # CHOOSE ONE.

```suggestion
    "URL": "https://cyanic.tk"
```
if you want a redirect to cyanic.tk

```suggestion
    "CNAME": "cyanic.tk"
```
if you want a masked website.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/Cyanic.json (hunk 1 above)
Comment (Cyanic76) [reply domains/Cyanic.json]: I'm sorry about it. I added the https:// protocol.

**Review by MaskDuck — APPROVED** [review]
[sus power uninjected]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
file name should be lowercased, i notified it after the ci checks

**Review by MaskDuck — APPROVED** [review]
[OPTIONAL] star this repo pls =)

**Comment by Cyanic76** [comment]
Thank you so much!

**Comment by MaskDuck** [comment]
if you need help open a github issue or join our discord server

**Pull request merged**